monkey/infection_monkey/post_breach/actions/modify_shell_startup_files.py

```diff
@@ -27,7 +27,10 @@ def run(self):
                     False,
                 )
             ]
-        PostBreachTelem(self, results).send()
+        # `command` is empty here since multiple commands were run and the results
```

Comment:
I don't understand this comment. So `command` field is empty, but multiple commands were run? The commands were aggregated into results?

Reply:
Changed